### DISCUSSION TITLE
.unityweb extension

### DISCUSSION THREAD

**Discussion by stefanwerleman** [jeffreylanters/react-unity-webgl#188] [Help Wanted!]
Hello I'm trying to follow the directions on the npm page and import my unity game onto my react app. I noticed when I build the webgl game, I get the following files that are different from the filenames in the npm page:

![image](https://user-images.githubusercontent.com/54635363/119024618-467d6c80-b971-11eb-8cc5-d05ef9c3aa04.png)

I'm only getting one `.js` file and the other three end with a `.unityweb` extension.

I followed the instructions and implemented the `UnityContext` below:

![image](https://user-images.githubusercontent.com/54635363/119024461-146c0a80-b971-11eb-81a2-a505e0dff8d4.png)

If this is the appropriate way to implement this, then I should see my unity game inside the bordered box, but it's not showing.

![image](https://user-images.githubusercontent.com/54635363/119025178-dde2bf80-b971-11eb-8a3d-abcc9d0e19ae.png)

Any suggestions would be very helpful 👍

**Comment by jeffreylanters** [comment]
Hi! What version of the Module and Unity are you using?

**Reply by stefanwerleman** [reply]
Hello, I used version `8.3.9` of the module but I realized I was using `Unity 2019.4.19`. So installed version `7.1.10` of the module and changed my code to look like this:

![image](https://user-images.githubusercontent.com/54635363/119036688-045b2780-b97f-11eb-89e5-cbc962048aa3.png)

Here is the project directory:

![image](https://user-images.githubusercontent.com/54635363/119036643-f3aab180-b97e-11eb-9bc8-d7226177f414.png)

This is what my webpage looks like:

![image](https://user-images.githubusercontent.com/54635363/119036803-2ce32180-b97f-11eb-9d74-f39271c07069.png)

In the chrome dev tools you can see the `div` tag.

Also, when I hover around the Unity `div` I get this:

![image](https://user-images.githubusercontent.com/54635363/119036985-69af1880-b97f-11eb-8a78-ac28b32e34f7.png)

**Reply by jeffreylanters** [reply]
Thanks for your detailed answer. I noticed an error in your console, could you send a screenshot (if it's relevant) as well?
